stand/libsa/strdup.c

```diff
@@ -41,14 +41,13 @@ static char sccsid[] = "@(#)strdup.c	8.1 (Berkeley) 6/4/93";
 char *
 strdup(const char *str)
 {
-	size_t len;
 	char *copy = NULL;
+	size_t len;
 
 	if (str != NULL) {
-	    len = strlen(str) + 1;
-	    if ((copy = malloc(len)) == NULL)
-		return (NULL);
-	    memcpy(copy, str, len);
+		len = strlen(str) + 1;
+		if ((copy = (char *)malloc(len)) != NULL)
+			memcpy(copy, str, len);
```

Comment:
this whole file is just style(9) changes. The changes look good, but should be separate